frontend/src/app/main/data/workspace/modifiers.cljs

```diff
@@ -348,9 +348,12 @@
 ;; Rotation use different algorithm to calculate children modifiers (and do not use child constraints).
 (defn set-rotation-modifiers
   ([angle shapes]
-   (set-rotation-modifiers angle shapes (-> shapes gsh/selection-rect gsh/center-selrect)))
+   (set-rotation-modifiers angle shapes {}))
 
-  ([angle shapes center]
+  ([angle shapes {:keys [as-delta?] :or {as-delta? false}}]
```

Comment:
I don't understand this. If someone was calling this function with the 3-arity variant `(set-rotation-modifiers angle shapes center)`, the center will now be interpreted as an invalid options argument.